README.md

```diff
@@ -72,3 +72,9 @@ $ python -m unittest discover tests/
 When using **signac** as part of your work towards a publication, we would really appreciate that you acknowledge **signac** appropriately.
 We have prepared examples on how to do that [here](http://docs.signac.io/en/latest/acknowledge.html).
 **Thank you very much!**
+
+## NumFOCUS
+
+The signac framework is a NumFOCUS Affiliated Project.

```

Comment:
If we keep this section, I would prefer a link here: `[NumFOCUS Affiliated Project](https://numfocus.org/sponsored-projects/affiliated-projects)`